core/src/main/java/org/infinispan/configuration/cache/StoreConfiguration.java

```diff
@@ -60,5 +60,10 @@ default boolean segmented() {
       return false;
    }
 
+   /**
+    * If true and the store support statistics, the statistic will be create during the startup.
```

Comment:
"The statistic will be created during the startup" sounds really specific, but this is a generic interface used by all stores, some of which may have statistics but not a `Statistics` object.